libraries/adaptive-expressions/src/localeInfo.ts

```diff
@@ -0,0 +1,916 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * locale info of decimal, thousands, grouping and currency.
+ */
+export const localeInfo = {
```

Comment:
It would be good to merge `main` in and make sure this runs through `eslint --fix`

Reply:
fix the linting